stdlib/public/Concurrency/GlobalExecutor.cpp

```diff
@@ -161,18 +161,6 @@ SerialExecutorRef _task_serialExecutor_getExecutorRef(
         const SerialExecutorWitnessTable *wtable);
 #pragma clang diagnostic pop
 
-// Implemented in Swift because we need to obtain the user-defined flags on the executor ref.
-//
-// We could inline this with effort, though.
-#pragma clang diagnostic push
-#pragma clang diagnostic ignored "-Wreturn-type-c-linkage"
-extern "C" SWIFT_CC(swift)
-TaskExecutorRef _task_executor_getTaskExecutorRef(
-    HeapObject *executor, const Metadata *selfType,
-    const SerialExecutorWitnessTable *wtable);
-#pragma clang diagnostic pop
```

Comment:
This was never used tbh, moving it and renamed since nothing called it before (ever) that's safe